explorer/client/src/app/App.tsx

```diff
@@ -1,8 +1,4 @@
-// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
This is from a pr merged this morning. I think you might need to rebase?

Reply:
from main ? 
i checked a new branch out from main and brought over all the explorer changes

Reply:
Yes: https://github.com/MystenLabs/sui/pull/1028. It was merged to main 7 hours ago